llvm/lib/Target/AIE/AIE2InstructionSelector.cpp

```diff
@@ -214,6 +215,8 @@ class AIE2InstructionSelector : public InstructionSelector {
   // Select a control packet header stream write instruction from the
   // corresponding intrinsic.
   bool selectPutMS_CPH(MachineInstr &I, MachineRegisterInfo &MRI);
+  bool selectBrCondLoopDecrement(MachineInstr &BrCond,
+                                 MachineRegisterInfo &MRI);
```

Comment:
Nit: Commit says it's cleaning something up, but I think it's only adding things now.

Reply:
Will fix.